lib/model/CModelDetailsView.cc

```diff
@@ -162,7 +162,7 @@ bool CModelDetailsView::hasByField() const {
     return (this->base().isPopulation()
                 ? this->base().dataGatherer().attributeFieldName()
                 : this->base().dataGatherer().personFieldName())
-        .empty();
+               .empty() == false;
```

Comment:
The other thing that might possibly improve readability is to split this into two statements.  Storing the result of the ternary operator in a `const std::string &` local variable and then returning a condition based on this in a second statement won't make the code less efficient once the optimiser has run on it, as the optimiser can easily see that the `const std::string &` can live in a register and never written back to memory.

And regardless of whether you decide to make that change, have you run `clang-format` on the file?

Reply:
Happy to split the statement. 

I have run it. It's `clang-format` that indented the line I changed. Not sure why. I imagine it will change again once I split the statement.